ci-operator/step-registry/gather/alibabacloud-console/gather-alibabacloud-console-commands.sh

```diff
@@ -0,0 +1,49 @@
+#!/bin/bash
+
+set -o nounset
+set -o errexit
+set -o pipefail
+
+trap 'CHILDREN=$(jobs -p); if test -n "${CHILDREN}"; then kill ${CHILDREN} && wait; fi' TERM
+
+
+
+if test -f "${KUBECONFIG}"
+then
+	oc --request-timeout=5s get nodes -o jsonpath --template '{range .items[*]}{.spec.providerID}{"\n"}{end}' | sed 's|.*/||' > "${TMPDIR}/node-provider-IDs.txt" &
+	wait "$!"
+
+	oc --request-timeout=5s -n openshift-machine-api get machines -o jsonpath --template '{range .items[*]}{.spec.providerID}{"\n"}{end}' | sed 's|.*/||' >> "${TMPDIR}/node-provider-IDs.txt" &
+	wait "$!"
+else
+	echo "No kubeconfig; skipping providerID extraction."
+fi
+
+if test -s "${SHARED_DIR}/alibaba-instance-ids.txt"
+then
+	cat "${SHARED_DIR}/alibaba-instance-ids.txt" >> "${TMPDIR}/node-provider-IDs.txt"
+else
+	echo "No alibaba-instance-ids.txt; skipping console log retrieval."
+    exit 0
```

Comment:
nit: inconsistent tab/space indent.  Also, in this case, we don't want to exit, because we may have content in `node-provider-IDs.txt` from the kubeconfig-powered `oc` listings.  Just logging the fact that `alibaba-instance-ids.txt` was missing/empty should be sufficient, and then we can continue on to install `aliyun`.  In the case where `node-provider-IDs.txt` turns out to be empty, the `aliyun` install is a waste, but :shrug: not so expensive that I think it needs a custom guard.

Reply:
Will check for `node-provider-IDs` before installing aliyun.